src/config/nunjucks.js

```diff
@@ -14,6 +14,19 @@ module.exports = {
       return translate(key, options);
     });
 
+    nunjucksEnv.addFilter("GDSDate", function (formatDate) {
+      let dateTransform = new Date(formatDate);
```

Comment:
Very minor but best practice to use `const` if you don't need to reassign the var

Reply:
Ah yes, I can see `dateFormat` could change but `dateTransform` doesn’t – I probably got confused because I’m using a method on `dateTransform`. I'll merge this and then update the code for the second part of `ipv-reuse`.